storm-client/src/jvm/org/apache/storm/daemon/worker/WorkerState.java

```diff
@@ -20,6 +20,17 @@
 
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Sets;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.*;
```

Comment:
I believe storm community says better to not use *. 

Btw, in Intellij, you can navigate Preference--> Editor --> code style --> Java and set `class count to use import with *` to a high number to avoid auto collapse

Reply:
That was just a re-order.

Reply:
Might be better to fix once we found it, but this is a kind of nit.